app/assets/javascripts/editor/editor.js.erb

```diff
@@ -1043,6 +1058,15 @@ var CodeOceanEditor = {
           delta_data.text === last_delta.text;
     },
 
+    showUserIsOnline: function (status_and_user) {
```

Comment:
The name of this method seems somewhat misleading, since it could also show that a user is offline.